specs/Media/mediasource-schema.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "id": "https://fiware.github.io/dataModels/Media/mediasource-schema.json",
+  "title": "FIWARE - Media / MediaSource",
+  "description": "Source for media events",
+  "definitions":{
+    "MediaSource": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "creationTime": {
+          "type": "string"
+        },
+        "sendTagsInEvents": {
+          "type": "boolean"
+        },
+        "parent":{ 
+          "$ref": "#/definitions/MediaSource" 
+        }
+      }
+    }
+  }
+}
```

Comment:
what's the purpose of this schema?

Reply:
The objective of this schema is to make generic the "mediasource", and facilitate the recursivity of the element itself. As an event can be generated after a indeterminated of media processes chained where the ouput of one of the filters is the "mediasource" of the next, going back until the "raw" origin. This field is used to track the filtering chain that leaded to the event.